src/ZODB/DB.py

```diff
@@ -385,23 +366,53 @@ def __init__(self, storage,
                  **storage_args):
         """Create an object database.
 
-        :Parameters:
-          - `storage`: the storage used by the database, e.g. FileStorage
-          - `pool_size`: expected maximum number of open connections
-          - `cache_size`: target size of Connection object cache
-          - `cache_size_bytes`: target size measured in total estimated size
-               of objects in the Connection object cache.
-               "0" means unlimited.
-          - `historical_pool_size`: expected maximum number of total
+        :param storage: the storage used by the database, such as a
+             :class:`~ZODB.FileStorage.FileStorage.FileStorage`.
+             This can be a string path name to use a constructed
+             :class:`~ZODB.FileStorage.FileStorage.FileStorage`
+             storage or ``None`` to use a constructed
+             :class:`~ZODB.MappingStorage.MappingStorage`.
+        :param int pool_size: expected maximum number of open connections.
+             Warnings are logged when this is exceeded and critical
+             messages are logged if twice the pool size is exceeded.
+        :param seconds pool_timeout: Maximum age of inactive connections
+             When a connection has remained unused in a connection
+             pool for more than pool_timeout seconds, it will be
+             discarded and it's resources released.
+        :param objects cache_size`: target maximum number of non-ghost
+             objects in each connection object cache.
+        :param int cache_size_bytes`: target total memory usage of non-ghost
+             objects in each connection object cache.
+        :param int historical_pool_size`: expected maximum number of total
             historical connections
-          - `historical_cache_size`: target size of Connection object cache for
-            historical (`at` or `before`) connections
-          - `historical_cache_size_bytes` -- similar to `cache_size_bytes` for
-            the historical connection.
-          - `historical_timeout`: minimum number of seconds that
-            an unused historical connection will be kept, or None.
-          - `xrefs` - Boolian flag indicating whether implicit cross-database
-            references are allowed
+        :param objects historical_cache_size`: target maximum number
+             of non-ghost objects in each historical connection object
+             cache.
+        :param int historical_cache_size_bytes`: target total memory
```

Comment:
There's a stray backtick (`) at the end of several param names, leading to odd rendering.